### PR TITLE
Updated payment endpoint to onlinepayment

### PR DESCRIPTION
**Pull request by MW-Forster**

### CODE REVIEW

File: src/EPR.Payment.Portal.Common/Constants/UrlConstants.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
     public static class UrlConstants
     {
         // Payments facade endpoints
-        public const string PaymentsComplete = "payments/{externalPaymentId}/complete";
-        public const string PaymentsInitiate = "payments";
+        public const string OnlinePaymentsComplete = "onlinepayments/{externalPaymentId}/complete";
```
Comment (davidjayakumarEviden) [inline src/EPR.Payment.Portal.Common/Constants/UrlConstants.cs:6]: change to online-payments and in the next line

**Review by davidjayakumarEviden — COMMENTED** [review]
Just one comment

File: src/EPR.Payment.Portal.Common/Constants/UrlConstants.cs (hunk 1 above)
Comment (MW-Forster) [reply src/EPR.Payment.Portal.Common/Constants/UrlConstants.cs:6]: Updated

**Review by davidjayakumarEviden — APPROVED** [review]
Looks good

**Pull request closed without merging**